modules/scf/Makefile

```diff
@@ -38,7 +38,7 @@ scf-brats-setup:
 	./brats_setup.sh
 
 .PHONY: upgrade
-upgrade:
+upgrade: chart gen-config
```

Comment:
Just noticed, this also breaks other workflow, IMHO upgrade should just upgrade, and do anything else. Maybe move this to a separate private target?

Reply:
I really wonder, because upgrading will by definition always use a different chart and/or registry creds than the install already present, so we need to recompute both just in case people are passing the options.
I'm thinking, but can't find another workflow right now.

Reply:
well, one that comes up to me is: one can deploy, and right after call upgrade to add an env var by modifying the values file manually, no need to uncompress the chart or regenerate the config in such cases.